### PR TITLE
Update C5G7 blueprints from reference

### PR DESCRIPTION
**Pull request by AidanMcDonald**
## What is the change?

Update a few numbers in C5G7 blueprints to match the reference document.

These specs are from Table 2 on page 6 of the composition/dimensions reference at [https://www.oecd-nea.org/upload/docs/application/pdf/2020-01/nsc-doc96-02-rev2.pdf](https://www.oecd-nea.org/upload/docs/application/pdf/2020-01/nsc-doc96-02-rev2.pdf):
1. Boron density in the moderator is specified as 2.78e-5 rather than 2.87e-5
2. Aluminum clad is composed of Al-27 rather than natural Aluminum
3. Fission chamber/central guide tube is stated to contain moderator plus 1.0e-8 atom/(b cm) U-235 (above Table 2 on page 6).

## Why is the change being made?

I happened to catch a couple minor transcription errors between the C5G7 blueprints in ARMI and the reference while running the C5G7 benchmark problem with the openmc plugin.

---

## Checklist

- [x] This PR has only [one purpose or idea](https://terrapower.github.io/armi/developer/tooling.html#one-idea-one-pr).
- [x] [Tests](https://terrapower.github.io/armi/developer/tooling.html#test-it) have been added/updated to verify any new/changed code.

- [x] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [x] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [x] The [release notes](https://terrapower.github.io/armi/developer/tooling.html#add-release-notes) have been updated if necessary.
- [x] The [documentation](https://terrapower.github.io/armi/developer/tooling.html#document-it) is still up-to-date in the `doc` folder.
- [x] The dependencies are still up-to-date in `pyproject.toml`.

### CODE REVIEW

File: armi/tests/tutorials/c5g7-blueprints.yaml [hunk 1]
```diff
@@ -65,9 +65,8 @@ custom isotopics:
         input format: number densities
         H: 6.70e-2
         O: 3.35E-2
-        B: 2.87E-5
-        U235: 6.0e-8
-        U238: 6.0e-9
```
Comment (jakehader) [inline armi/tests/tutorials/c5g7-blueprints.yaml:70]: Although the reference states 1.0e-8 atom/b-cm, does the comment of ~90% enrichment still apply? If so, can we add U-238 at this fraction or remove the comment?

Comment (john-science) [reply armi/tests/tutorials/c5g7-blueprints.yaml:70]: >  I will check it out in more depth.

Can't wait!

**Comment by AidanMcDonald** [comment]
Found another change: temperature is specified as 20C.

**Comment by john-science** [comment]
> Found another change: temperature is specified as 20C.

@AidanMcDonald Any updates for us?

**Comment by AidanMcDonald** [comment]
@john-science I have been working to get the correct benchmark results and finally succeeded, but the case that replicates the benchmark results is sufficiently different (the benchmark problem blends the fuel and clad together) that I think we shouldn't update the tutorial case to match it. My plan is to have the simpler exact benchmark case live in the openmc plugin and have a separate entry point in the plugin to run this theoretically more accurate case. I will have everything cleaned up and pushed in the coming days.

TL;DR: No update yet, but soon.

**Comment by AidanMcDonald** [comment]
I've gone over the reference with a fine-toothed comb and can't find the source of the 90% enrichment number quoted in the comment. I'm now inclined to believe it was just a reasonable assumption like the comment said. I have replaced the old comment with a new one quoting the source of the new number.

**Review by john-science — APPROVED** [review]
I like it!

Thanks, @AidanMcDonald 

Before I go ahead and merge this, one last check: Do you think it's ready for prime time?

**Comment by AidanMcDonald** [comment]
Yes. I'm finally done fiddling with this and the homogenized "official" benchmark in the openmc plugin. It's good to go.

**Review by john-science — APPROVED** [review]
